matroska_tags.xml

```diff
@@ -27,7 +27,7 @@
       <description lang="en">A tag that contains other tags to describe a sample used in the targeted item taken from another work of art. All tags in this list can be used "under" the SAMPLE tag like TITLE, ARTIST, DATE_RELEASED, etc.</description>
     </tag>
     <tag name="COUNTRY" class="Nesting Information" type="UTF-8">
-      <description lang="en">The name of the country ([biblio ISO-639-2](http://lcweb.loc.gov/standards/iso639-2/englangn.html#two)) that is meant to have other tags inside (using nested tags) to country specific information about the item. All tags in this list can be used "under" the COUNTRY_SPECIFIC tag like LABEL, PUBLISH_RATING, etc.</description>
+      <description lang="en">The name of the country ([biblio ISO-639-2](https://lcweb.loc.gov/standards/iso639-2/englangn.html#two)) that is meant to have other tags inside (using nested tags) to country specific information about the item. All tags in this list can be used "under" the COUNTRY_SPECIFIC tag like LABEL, PUBLISH_RATING, etc.</description>
```

Comment:
https version gives "Http/1.1 Service Unavailable", temporary or just not yet ready?

Reply:
Hmm… on my end still redirects `http` to `https`.

Reply:
![image](https://user-images.githubusercontent.com/1641638/58379096-630f8580-7f9f-11e9-9fe3-89323afae69b.png)

Reply:
I guess here is the hint:

<img width="763" alt="Screenshot 2019-05-26 at 10 42 33" src="https://user-images.githubusercontent.com/3246607/58379393-fe562a00-7fa2-11e9-9be8-728f922687cc.png">

Should I change to `https://www.loc.gov/standards/iso639-2/php/English_list.php`?

Reply:
OK.

Reply:
done